patches/common/chromium/dcheck.patch

```diff
@@ -3,12 +3,12 @@ index 582fb89868cd..24d3e82232b6 100644
 --- a/base/logging.h
 +++ b/base/logging.h
 @@ -850,7 +850,7 @@ const LogSeverity LOG_DCHECK = LOG_INFO;
- 
+
```

Comment:
All those whitespace shouldn't have been removed =/
@codebytere

Reply:
yeah i know, i have literally no clue how that happened but i'll fix it